ProjectsJenkinsfile

```diff
@@ -90,26 +90,13 @@ def dub_projects = [
 
 node { // for now whole pipeline runs on one node because no slaves are present
 
-    stage('Get compiler') {
-        unstash name: "dlang-build"
-    }
-
-    stage('Clone') {
-        parallel mapSteps(dub_projects, this.&clone)
-    }
-
-    stage('Build') {
-        parallel mapSteps(dub_projects, { name ->
-            withEnv(["PATH=${env.WORKSPACE}/distribution/bin:${env.PATH}"]) {
-                sh "dub build"
-            }
-        })
-    }
+    unstash name: "dlang-build"
 
-    stage('Test') {
+    stage('Test Projects') {
         parallel mapSteps(dub_projects, { name ->
+            clone(name)
             withEnv(["PATH=${env.WORKSPACE}/distribution/bin:${env.PATH}"]) {
-                sh "dub build"
+                sh "dub test"
```

Comment:
What's supposed to be tested by that? Building without `-unittest`? Wouldn't that always be a subset?
Remember we're not testing dub projects here, but dmd/druntime/phobos.